android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UrlDeviceDiscoveryService.java

```diff
@@ -211,6 +211,7 @@ public IBinder onBind(Intent intent) {
 
   @Override
   public boolean onUnbind(Intent intent) {
+    triggerCallback();
```

Comment:
This shouldn't be needed...any update that has come through should already be communicated.